CMakeLists.txt

```diff
@@ -260,6 +261,20 @@ IF(WANT_PORTAUDIO)
 	ENDIF(PORTAUDIO_FOUND)
 ENDIF(WANT_PORTAUDIO)
 
+# check for libsoundio
+IF(WANT_SOUNDIO)
+  FIND_PACKAGE(SoundIo)
+  IF(SOUNDIO_FOUND)
+    SET(LMMS_HAVE_SOUNDIO TRUE)
+    SET(STATUS_SOUNDIO "OK")
+    INCLUDE_DIRECTORIES("${SOUNDIO_INCLUDE_DIR}")
```

Comment:
> grepped for portaudio, copied all the cmake configuration, then replaced portudio with soundio

That's what I get for only reading the comments.  :smile: 

I see now that it's a single line... `INCLUDE_DIRECTORIES`.  Did you clone the `IF(NOT ("${PULSEAUDIO_INCLUDE_DIR}" STREQUAL ""))` logic?  Because if you don't define that, you're better off doing an `IF(WANT_SOUNDIO AND SOUNDIO_FOUND)`, right?